src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -328,13 +329,25 @@ public function url($path)
      *
      * @param  \League\Flysystem\AwsS3v3\AwsS3Adapter  $adapter
      * @param  string  $path
+     * @param  string  $expires
      * @return string
      */
-    protected function getAwsUrl($adapter, $path)
+    protected function getAwsUrl($adapter, $path, $expires)
```

Comment:
Good, there something else 
the method ``getAwsUrl`` is only used inside this class and just in this case so it wont break anything.